### PR TITLE
Basic map reduce for create dataset properties

### PR DESCRIPTION
**Pull request by asbjrnmunk**

### CODE REVIEW

**Comment by asbjrnmunk** [comment]
Sure!

I have tested it on:

1. Unsupervised and supervised dataset
2. With 1 and 12 workers/processes

I dont have any 2D dataset right now, so i cannot compare with a previous dataset properties.

**Comment by Sllambias** [comment]
Super! I have a 2D dataset so I can test that one

File: yucca/planning/dataset_properties.py [hunk 1]
```diff
@@ -56,66 +59,100 @@ def create_properties_pkl(data_dir, save_dir, suffix=".nii.gz"):
...
+            logging.warn(
+                f"A volume has more than three dimensions. This is most often a mistake." f"Dims: {dim}, Vol: {subject}"
+            )
+
+        size = image.shape
+
+        if isinstance(image, nib.Nifti1Image):
+            spacing = get_nib_spacing(image).tolist()
+        else:
+            spacing = [1.0, 1.0, 1.0]
```
Comment (Sllambias) [inline yucca/planning/dataset_properties.py:142]: (not your fault, my own that I just spotted now that you changed the code) change this to "[1.0] * dim" to also support 2D images

**Review by Sllambias — APPROVED** [review]
One tiny comment (that is not even your mistake) otherwise lgtm

**Pull request merged**